app-frontend/config/webpack.common.config.js

```diff
@@ -0,0 +1,68 @@
+// Modules
+var webpack = require('webpack'),
+    autoprefixer = require('autoprefixer'),
+    HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+    entry: {
+        app: __dirname + '/../src/app/app.js'
```

Comment:
There should be a separate vendor bundle that we don't run through our minifier and where possible is pulled from `node_modules`.

I think this [generator](https://www.npmjs.com/package/generator-angular-webpack-es6) does a pretty good job setting this stuff up from the beginning and has a pretty fast build time.